### PR TITLE
[Go]: Add hertz

### PR DESCRIPTION
**Pull request by FlameMida**

### CODE REVIEW

**Comment by waghanza** [comment]
Hi,

Thanks for your submission. 

We require explicit consent (from authors / active contributors) to include any framework in here. As I can see you are not part of @cloudwego org.

Do you want to include `hertz` in here @baiyutang @PureWhiteWu @Duslia  @GuangmingLuo ?

File: go/hertz/config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+framework:
+  website: https://www.cloudwego.io/docs/hertz/
```
Comment (waghanza) [inline go/hertz/config.yaml:2]:
```suggestion
  website: cloudwego.io/docs/hertz/
```

File: go/hertz/config.yaml [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+  version: 0.3.0
```
Comment (waghanza) [inline go/hertz/config.yaml:3]:
```suggestion
  version: 0.3
```

File: go/hertz/go.mod [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+	github.com/henrylee2cn/goutil v0.0.0-20210127050712-89660552f6f8 // indirect
+	github.com/klauspost/cpuid/v2 v2.0.9 // indirect
+	github.com/nyaruka/phonenumbers v1.0.55 // indirect
+	github.com/tidwall/gjson v1.13.0 // indirect
+	github.com/tidwall/match v1.1.1 // indirect
+	github.com/tidwall/pretty v1.2.0 // indirect
+	github.com/twitchyliquid64/golang-asm v0.15.1 // indirect
+	golang.org/x/arch v0.0.0-20210923205945-b76863e36670 // indirect
+	google.golang.org/protobuf v1.27.1 // indirect
+)
```
Comment (waghanza) [inline go/hertz/go.mod:22]:
```suggestion
require github.com/cloudwego/hertz v0.3.0
```

**Review by waghanza — CHANGES_REQUESTED** [review]

**Review by waghanza — APPROVED** [review]

File: go/hertz/go.mod [hunk 4]
```diff
@@ -1,22 +1,3 @@
 module main
 
 require github.com/cloudwego/hertz v0.3.0
-
-require (
-	github.com/bytedance/go-tagexpr/v2 v2.9.2 // indirect
-	github.com/bytedance/gopkg v0.0.0-20220413063733-65bf48ffb3a7 // indirect
-	github.com/bytedance/sonic v1.4.0 // indirect
```
Comment (FlameMida) [inline go/hertz/go.mod]: In GO 1.19 should replace sonic with 1.4 version

Comment (waghanza) [reply go/hertz/go.mod]: worth to subit a patch on `hertz` side imho 

cc @Duslia @FGYFFFF

Comment (Duslia) [reply go/hertz/go.mod]: OK we will update it soon

File: go/hertz/go.mod [hunk 5]
```diff
@@ -0,0 +1,3 @@
+module main
+
+require github.com/cloudwego/hertz v0.3.0
```
Comment (waghanza) [inline go/hertz/go.mod:3]:
```suggestion
require github.com/cloudwego/hertz v0.3.1
```

**Review by waghanza — CHANGES_REQUESTED** [review]

**Review by waghanza — APPROVED** [review]

**Comment by waghanza** [comment]
`hertz` is now included in https://web-frameworks-benchmark.netlify.app/result?f=hertz

@FlameMida  @baiyutang @PureWhiteWu @Duslia @GuangmingLuo

**Pull request merged**